app/views/govuk_publishing_components/components/docs/cards.yml

```diff
@@ -0,0 +1,143 @@
+        - link:
+            text: Business and self-employed
+            path: http://www.gov.uk
+          description: Tools and guidance for businesses
+        - link:
+            text: Business and self-employed
+            path: http://www.gov.uk
+          description: Tools and guidance for businesses
+        - link:
+            text: Business and self-employed
+            path: http://www.gov.uk
+          description: Tools and guidance for businesses
+  two_column_layout:
+    description:  Override default three column layout on desktop by passing through `two_column_layout` parameter (defaults to `false`).
+    data:
+      two_column_layout: true
+      items:
+        - link:
+            text: Benefits
+            path: http://www.gov.uk
+          description: Includes eligibility, appeals, tax credits and Universal Credit
+        - link:
+            text: Births, deaths, marriages and&nbsp;care
+            path: http://www.gov.uk
+          description: Parenting, civil partnerships, divorce and Lasting Power of Attorney</p>
+        - link:
+            text: Business and self-employed
+            path: http://www.gov.uk
+          description: Tools and guidance for businesses
+        - link:
+            text: Childcare and parenting
+            path: http://www.gov.uk
+          description: Includes giving birth, fostering, adopting, benefits for children, childcare and schools
+        - link:
+            text: Citizenship and living in the&nbsp;UK
+            path: http://www.gov.uk
+          description: Voting, community participation, life in the UK, international projects
+        - link:
+            text: Crime, justice and the&nbsp;law
+            path: http://www.gov.uk
+          description: Legal processes, courts and the police
+        - link:
+            text: Business and self-employed
+            path: http://www.gov.uk
+          description: Tools and guidance for businesses
+        - link:
+            text: Business and self-employed
+            path: http://www.gov.uk
+          description: Tools and guidance for businesses
+        - link:
+            text: Business and self-employed
+            path: http://www.gov.uk
+          description: Tools and guidance for businesses
+        - link:
+            text: Business and self-employed
+            path: http://www.gov.uk
+          description: Tools and guidance for businesses
+  custom_heading_levels:
+    description: |
+      Override default heading level by passing through `heading_level` parameter (defaults to `<h2>`).
+
+      Override default subheading level by passing through `sub_heading_level` parameter (defaults to `<h3>`)
+    data:
+      heading: Topics
+      heading_level: 3
+      sub_heading_level: 4
+      items:
+        - link:
+            text: Benefits
+            path: http://www.gov.uk
+          description: Includes eligibility, appeals, tax credits and Universal Credit
+  with_tracking:
```

Comment:
Usually we try to avoid examples that explicitly describe themselves as adding tracking, partly because that can raise privacy concerns but also because this doesn't appear to actually add tracking - the component would still need to be wrapped in a container with the `gem-track-click` module applied for tracking to occur. Maybe `with_link_data_attributes` instead?

Reply:
We already use 'With tracking' on other component examples like https://components.publishing.service.gov.uk/component-guide/select/with_tracking. But happy to go with your suggestion, it seems sensible 👍

Reply:
Yes, some of our code doesn't follow our current best practice unfortunately. I'll make a note to find some time to clean up this and any other examples I can find, thanks for finding that one.

Reply:
I see you've renamed this but it still explicitly mentions tracking, which I'd like to avoid. For preference I think examples like this (and the code it refers to) should use the terminology of `data_attributes` rather than `tracking_attributes`. 

I appreciate that it's a pain to rename it at this point and there may be other components that don't follow this convention but I'm going to be looking into this soon to try to sort out any strays from this convention, and changing this now before it is put into use is going to be much easier than changing it later.

Reply:
We could look at this as part of the card for the discussion here: https://github.com/alphagov/govuk_publishing_components/pull/2624#discussion_r817495300

I've also just added a description to this example to explain that no tracking events fire within the component guide.